### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by oyelakinG9**
- Create a new GitHub repo... setup
- Create a new branch according to the GitHub flow rules.
- Add a .gitignore file.
- Set up a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
- Add a descriptive README file to your project - please use this template but remember to customize it to your project.
- Add 3 files (each one in a separate commit): 
A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
A .css file that includes one class with styles for your header (make it in your favorite color).
A test.md file with the text "This file should be ignored by git".

- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.
Open a pull request. 
This project based on the requirements above

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
```
Comment (alonsofalconi) [inline README.md]: - [x] You have done an excellent job with your project but I would suggest removing the sections that are not in use to make your README look even more professional :raised_hands:
image

**Review by alonsofalconi — CHANGES_REQUESTED** [review]
Hi @oyelakinG9,

You have done a great job so far! :muscle:
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## To highlight :dart:
- Linters working correctly :heavy_check_mark:
- Well documented PR title and description :heavy_check_mark:
- Commit messages are meaningful :heavy_check_mark:
- GitHub flow :heavy_check_mark:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Capitalize the first letter of your commit messages :wink:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oyelakinG9** [comment]
Hi Alonso,
Many works have been done and corrected.
This is a new modified request.
Thanks.

**Review by rustam-tolipov — APPROVED** [review]
## Hi @oyelakinG9  👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

niceJob

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**